.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,8 +13,8 @@ labels: bug
 - `:checkhealth` result
```

Comment:
should be `nvim-lspconfig` now

Reply:
checkhealth result can be long, we should provide one those markdown blocks you can fold.

Reply:
it's `:checkhealth lspconfig`, updated